client/my-sites/earn/memberships/add-edit-plan-modal.jsx

```diff
@@ -0,0 +1,401 @@
+					</FormLabel>
+					<FormTextInput
+						id="title"
+						value={ editedProductName }
+						onChange={ onNameChange }
+						onBlur={ () => setFocusedName( true ) }
+					/>
+					{ ! isFormValid( 'name' ) && focusedName && (
+						<FormInputValidation isError text={ translate( 'Please input a name.' ) } />
+					) }
+				</FormFieldset>
+				<FormFieldset>
+					<FormToggle onChange={ handlePayWhatYouWant } checked={ editedPayWhatYouWant }>
+						{ translate( 'Enable customers to pick their own amount ("Pay what you want").' ) }
+					</FormToggle>
+				</FormFieldset>
+				<FormFieldset>
+					<FormToggle onChange={ handleMultiplePerUser } checked={ editedMultiplePerUser }>
+						{ translate( 'Allow the same customer to sign up multiple times to the same plan.' ) }
+					</FormToggle>
+				</FormFieldset>
+			</>
+		);
+	};
+
+	const renderEmailTab = () => {
+		return (
+			<>
+				<FormFieldset>
+					<h6 className="memberships__dialog-form-header">{ translate( 'Posts via email' ) }</h6>
+					<p>
+						{ translate(
+							'Allow members of this recurring payment plan to opt into receiving new posts via email.'
+						) }{ ' ' }
+						<InlineSupportLink
+							supportPostId={ 154624 }
+							supportLink="https://wordpress.com/support/wordpress-editor/blocks/recurring-payments-button/" // TODO: Link to specific section once article is update
+							showIcon={ false }
+							text={ translate( 'Learn more.' ) }
+						/>
+					</p>
+					<FormToggle
+						onChange={ ( newValue ) => setEditedPostsEmail( newValue ) }
+						checked={ editedPostsEmail }
+					>
+						{ translate(
+							'Email newly published posts to members of this recurring payment plan who have opted in.'
+						) }
+					</FormToggle>
+				</FormFieldset>
+				<FormFieldset>
+					<h6 className="memberships__dialog-form-header">
+						{ translate( 'Custom confirmation message' ) }
+					</h6>
+					<p>
+						{ translate(
+							'Add a custom message to the confirmation email that is sent out for this recurring payment plan.'
+						) }
+					</p>
+					<CountedTextArea
+						value={ editedCustomConfirmationMessage }
+						onChange={ ( event ) => setEditedCustomConfirmationMessage( event.target.value ) }
+						acceptableLength={ MAX_LENGTH_CUSTOM_CONFIRMATION_EMAIL_MESSAGE }
+						showRemainingCharacters={ true }
+						placeholder={ translate( 'Thank you for subscribing!' ) }
+					/>
+				</FormFieldset>
+			</>
+		);
+	};
+
+	return (
+		<Dialog
+			isVisible={ isVisible }
+			onClose={ onClose }
+			buttons={ [
+				{
+					label: translate( 'Cancel' ),
+					action: 'cancel',
+				},
+				{
+					label: product ? translate( 'Edit' ) : translate( 'Add' ),
+					action: 'submit',
+					disabled: ! isFormValid(),
+					isPrimary: true,
+				},
+			] }
+		>
+			<FormSectionHeading>
+				{ product
+					? translate( 'Edit Recurring Payment plan' )
+					: translate( 'Add new Recurring Payment plan' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Add New Recurring Payment plan' )` *ES Score: 11*